### PR TITLE
FV_klahoose - new desktop keyboard

### PR DESCRIPTION
**Pull request by xyzhelen**
new desktop keyboard, new desktop help, minor updates to mobile
@caforbes

### CODE REVIEW

**Comment by keyman-server** [comment]
Thank you for your pull request. The Keyman keyboard review team have been notified of your pull request and will review it and build it shortly.

**Comment by LornaSIL** [comment]
I guess we are waiting for @caforbes to check this before we do? 

Also, the branch has conflicts so it would be good to go ahead and resolve those. 

From March 1-16 we won't be doing much keyboard checking, so if you want this done next week, please let us know when it's ready.

File: release/fv/fv_klahoose/source/fv_klahoose.kmn [hunk 1]
```diff
@@ -115,3 +117,120 @@ store(&LAYOUTFILE) "fv_klahoose.keyman-touch-layout"
...
+C store(acutekey) '`'
+any(vowels) + any(acutekey) > index(vowels_acute,1)
+
+C accents (2) - accute, glottal comma above
++ "\" > "̌"   c cdmcar caron = backslash
++ "`" > "́"    c cdmacu acute = backtick         
++ "'" > "̓"    c cdmeje combining comma above= apostrophe
+
+C provide a way to type 'ɑ' (TODO: is this needed?)
+C "æ" + "A" > 'ɑ' C user holds SHIFT and types "a" twice
```
Comment (caforbes) [inline release/fv/fv_klahoose/source/fv_klahoose.kmn:174]: I don't see this combo in the documentation, could you make sure that it is added there?

File: release/fv/fv_klahoose/source/fv_klahoose.kmn [hunk 2]
```diff
@@ -115,3 +117,120 @@ store(&LAYOUTFILE) "fv_klahoose.keyman-touch-layout"
...
++ [SHIFT K_X] > 'χ'
++ [RALT K_X] > 'X'
+
++ [SHIFT K_V] > 'w̥'
++ [RALT K_V] > 'V'
+
++ [K_SLASH] > 'ʔ'
++ [RALT K_SLASH] > '/'
+
+c quotes
```
Comment (caforbes) [inline release/fv/fv_klahoose/source/fv_klahoose.kmn:214]: Could we please remove the curly quotes and guillemets from the keyboard? I think having the regular `"` symbol available, and regular `'` on rightalt, is fine. We can add them back if we hear that they are important, I would prefer to add later than remove later.

File: release/fv/fv_klahoose/source/fv_klahoose.kvks [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    <layer shift="SRA">
+      <key vkey="K_0">›</key>
+      <key vkey="K_9">‹</key>
+    </layer>
+    <layer shift="">
+      <key vkey="K_QUOTE">◌̓</key>
+      <key vkey="K_SLASH">ʔ</key>
+      <key vkey="K_COLON">ə</key>
+      <key vkey="K_LBRKT">ƛ</key>
+      <key vkey="K_BKSLASH">◌̌</key>
```
Comment (caforbes) [inline release/fv/fv_klahoose/source/fv_klahoose.kvks:43]: I have heard that the dashed circle thing doesn't display well on all platforms, so we may need to use the standalone version of the key (e.g. non-combining hachek) in the onscreen keyboard. The Keyman team may be able to confirm.

**Review by caforbes — COMMENTED** [review]
I noticed just a few things that we may want to revise, in addition to the merge conflict that still needs fixing.

**Comment by DavidLRowe** [comment]
I'd suggest that you not worry about the "conflicts" message for the time being. If you fix the things that have been mentioned, one of us (@LornaSIL or me) will start a build. Often I've seen that that message goes away once the build is complete. If there is a true conflict, we can deal with it then.

File: release/fv/fv_klahoose/source/fv_klahoose.kmn (hunk 1 above)
Comment (caforbes) [reply release/fv/fv_klahoose/source/fv_klahoose.kmn:174]: Oops, missed that. All good then!

**Comment by keyman-server** [comment]
Thank you for your pull request. The Keyman keyboard review team have been notified of your pull request and will review it and build it shortly.

File: release/fv/fv_klahoose/source/fv_klahoose.kmn (hunk 2 above)
Comment (xyzhelen) [reply release/fv/fv_klahoose/source/fv_klahoose.kmn:214]: done

**Comment by keyman-server** [comment]
Thank you for your pull request. The Keyman keyboard review team have been notified of your pull request and will review it and build it shortly.

**Review by DavidLRowe — APPROVED** [review]

**Pull request merged**